CHANGELOG.md

```diff
@@ -2,6 +2,42 @@ Changes by Version
 ==================
 <!-- next version -->
 
+## 0.93.0
+
+### 🛑 Breaking changes 🛑
+
+- `autoinstrumentation`: Bump OpenTelemetry .NET Automatic Instrumentation to 1.3.0 (#2538)
```

Comment:
of note – we do not automatically upgrade users to this version. The version is still set to 1.2.0